lib/modules/dosomething/dosomething_kudos/dosomething_kudos.install

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @file
+ * Installation and schema hooks for dosomething_kudos.module
+ */
+
+/**
+ * Implements hook_schema().
+ */
+function dosomething_kudos_schema() {
+  $schema = array();
+  $schema['dosomething_kudos'] = array(
+    'description' => 'Table of kudos interactions on reportback items.',
+    'fields' => array(
+      'kid' => array(
+        'description' => 'The primary identifier for a kudos record.',
+        'type' => 'serial',
+        'not null' => TRUE,
+      ),
+      'fid' => array(
+        'description' => 'The {reportback_file}.fid that this kudos applies to.',
+        'type' => 'int',
+        'not null' => TRUE,
+        'default' => 0,
+      ),
+      'uid' => array(
+        'description' => 'The {users}.uid that applied the kudos.',
+        'type' => 'int',
+        'not null' => TRUE,
+        'default' => 0,
+      ),
+      'tid' => array(
+        'description' => 'The {taxonomy_term}.tid that this kudos belongs to.',
+        'type' => 'int',
+        'not null' => TRUE,
+        'default' => 0,
+      ),
+    ),
+    'primary key' => array('kid'),
+    'indexes' => array(
+      'fid' => array('fid'),
+      'uid' => array('uid'),
+      'tid' => array('tid'),
```

Comment:
Just a sanity check: the `tid` index is intentional? For querying  all kudos records for a certain `tid`?